PhiferTristan/kneel-server
# view check on methods

Added conditions to json-server.py when performing a POST or DELETE that checks if "orders" string is in the path, and if not, give a status code 405 back and a message telling the user the method is not allowed. Attempting to use the PUT method at all returns a status code 405 and tells the user the method is not allowed.

## Steps to Test

1.Pull 'tp-sizes-view' branch and switch to it
2. Open Postman
3. Try a PUT request on any of the tables and verify response and code are telling the user it is not allowed
4. Try a POST or DELETE request at /metals, /sizes, or /styles and verify response and code are telling the user it is not allowed
